nautobot/docs/user-guide/administration/configuration/authentication/sso.md

```diff
@@ -546,7 +546,42 @@ SOCIAL_AUTH_AZUREAD_TENANT_OAUTH2_TENANT_ID = "<Tenant ID from Azure>"
 
 With those settings in place your users should be able to authenticate against Azure AD and successfully login to Nautobot. However, that user will not be placed in any groups or given any permissions. In order to do so, you'll need to utilize a script to synchronize the groups passed from Azure to Nautobot after authentication succeeds. Any group permissions will need to be set manually in the Nautobot admin panel.
 
-An example to sync groups with Azure is provided in the [`examples/azure_ad`](https://github.com/nautobot/nautobot/tree/main/examples/azure_ad) folder in the root of the Nautobot repository.
+A group syncing function is provided and but needs to be configured. See [Group Syncing](#group_syncing).
 
 !!! note
     You may need to set `UWSGI_BUFFER_SIZE` to something bigger than the default 4096 bytes in the UWSGI config if you are seeing errors like `invalid request block size` in your application logs (see [here](https://uwsgi-docs.readthedocs.io/en/latest/Options.html#buffer-size) for more information)
+
+## Group Syncing
+
+To do so you can add `nautobot.extras.group_sync` to `SOCIAL_AUTH_PIPELINE` like:
+
+```python
+SOCIAL_AUTH_PIPELINE = (
+    "social_core.pipeline.social_auth.social_details",
+    "social_core.pipeline.social_auth.social_uid",
+    "social_core.pipeline.social_auth.auth_allowed",
+    "social_core.pipeline.social_auth.social_user",
+    "social_core.pipeline.user.get_username",
+    "social_core.pipeline.user.create_user",
+    "social_core.pipeline.social_auth.associate_user",
+    "social_core.pipeline.social_auth.load_extra_data",
+    "social_core.pipeline.user.user_details",
+    "nautobot.extras.group_sync",
+)
+```
+
+You must then enable the scope that will provide your groups in your provider and then configure the scope to use:
+
+```python
+# for Azure
+NAUTOBOT_SSO_CLAIMS_GROUP = "roles"
```

Comment:
Do these settings need to be prefixed `NAUTOBOT_` to avoid collision with something built in to social-auth, or could they just be `SSO_CLAIMS_GROUP`, `SSO_SUPERUSER_GROUPS`, `SSO_STAFF_GROUPS`?

Reply:
They definitely can. I originally had this reading an env var in there and not using settings and bad copy and paste.